bundles/AdminBundle/Controller/Rest/Element/DataObjectController.php

```diff
@@ -125,10 +125,11 @@ public function getAction(Request $request, $id = null)
             $stopwatch->start('ws', $profileName);
         }
 
-        /** @var WebserviceObjectOut|WebserviceFolderOut $out */
         if ($object instanceof DataObject\Folder) {
+            /** @var WebserviceObjectOut|WebserviceFolderOut $out */
             $out = $this->service->getObjectFolderById($id);
         } else {
+            /** @var WebserviceObjectOut|Out $out */
```

Comment:
Not that the `use` statement is removed, this should be removed, too.

```suggestion
            /** @var WebserviceObjectOut $out */
```